packages/config/config/devices/0x027a/zse11.json

```diff
@@ -67,22 +79,160 @@
 			"label": "Flash LED on Motion",
 			"defaultValue": 1
 		},
+		{
+			"#": "20",
+			"$if": "firmwareVersion >= 2.0",
+			"label": "Vibration Sensor",
+			"description": "Vibration sensor enable or disable.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disable vibration sensor",
+					"value": 0
+				},
+				{
+					"label": "Enable vibration sensor",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "21",
+			"$if": "firmwareVersion >= 2.0",
+			"label": "Light Sensor Checking Interval",
+			"description": "Set the frequency interval for lux value verification.  This is how often the sensor will probe the environment for lux value changes. NOTE: a report will be sent ONLY if the value has changed by the threshold value set in Parameter 185 as compared to the most recent report.",
```

Comment:
Regarding this parameter and the following: Zooz parameter descriptions are terribly verbose and tend to repeat themselves in the label, description and option labels. We prefer keeping things short since they will be displayed on constrained UIs.

Ideally align these with how I did it in [this PR](https://github.com/zwave-js/node-zwave-js/pull/7496/files), so we can extract them into templates in a followup PR.